### PR TITLE
[TIDOC-2058] Added @editurl JSDuck field that contains GitHub edit link for that class/API

### PR DESCRIPTION
**Pull request by tstatler**
To support https://jira.appcelerator.org/browse/TIDOC-2058.

Note: This step is done automatically for the Titanium docs, since they need to converted from YAML to JSDuck format anyway. Alloy docs are in JSDuck format already, so they are passed as-is to JSDuck for processing. As there are only a few Alloy doc sources, it made sense just to add them manually.

### CODE REVIEW

File: docs/apidoc/backbone.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/**
+ * @class Backbone.Events
```
Comment (bhatfield) [inline docs/apidoc/backbone.js]: This is showing up in the TOC.  Maybe use `@pseudo` to hide?

**Comment by bhatfield** [comment]
@tstatler 
This looks fine except for Backbone.Event which is showing up in the TOC.

You will need Mr. Poulsen or somebody with merge permissions to commit this.

**Comment by tstatler** [comment]
@bhatfield , thanks, I removed that file from the PR.

**Comment by skypanther** [comment]
Passes all tests locally. 

**APPROVED**

**Pull request merged**